### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by zheng-xing** [kkweon/UNet-in-Tensorflow#8]
Thanks for your code. I think you defined the IOU loss function using the dice formula.
It would be clearer to rename the function as "dice_coef".
